docs/articles/sde/introduction_to_sde.rst

```diff
@@ -35,13 +36,24 @@ Following is a list of environment variables used by vagrant when provisioning
 the SDE.
 
 * SDK_FILE_NAME: Name of the self-extracting SDK package to install into the
-  SDE. Can include wildcards. Defaults to `oecore*toolchain*sh`.
+  SDE. Can include bash wildcards. Defaults to `oecore*toolchain*sh`.
```

Comment:
Might be out of scope but we should probably change the default value to `pelux*toolchain*sh`

Reply:
Hm yeah, or even "*toolchain*sh" in the script but also change the sdk-name from oecore... to pelux... I'll create a issue for that.

Reply:
https://github.com/Pelagicore/meta-pelux/issues/90